README.md

```diff
@@ -6,7 +6,7 @@
 
 ### Deprecation announcement
 
-Version 2.1.0 of this project ends support for Python 2.7, 3.3, and 3.4. We recommend migrating your projects to Python 3.5 or newer as soon as possible. We'll be happy to answer any questions you have in [a GitHub issue](https://github.com/logzio/logzio-python-handler/issues).
+Version 3.0.0 of this project ends support for Python 2.7, 3.3, and 3.4. We recommend migrating your projects to Python 3.5 or newer as soon as possible. We'll be happy to answer any questions you have in [a GitHub issue](https://github.com/logzio/logzio-python-handler/issues).
```

Comment:
I think we need to mention here the first version we stoped supporting Python2.7...
@imnotashrimp WDYT?

Reply:
@idohalevi  it is the first version, we still haven't put this new version in pyPI and I read the [Semantic Versioning doc](https://semver.org/) and realised that I made a mistake in the versioning, and that it should be 3.0.0 instead of 2.1.0 (major.minor.patch).